doc/content/resources/neams-workbench.md

```diff
@@ -0,0 +1,162 @@
+[NEAMS tools](https://neams.inl.gov/code-descriptions/) are also designed to be interoperable, allowing users to construct customized multiphysics workflows. This flexibility is achieved by using the Multiphysics Object-Oriented Simulation Environment ([MOOSE](https://mooseframework.inl.gov/index.html)) framework, which is the foundation of several solvers and also provides the functionality to couple individual applications in loose or tight coupling schemes through the [MOOSE MultiApp System](https://mooseframework.inl.gov/syntax/MultiApps/index.html).
+
+[The NEAMS Workbench](https://www.ornl.gov/onramp/neams-workbench) [!citep](Lefebvre19)  initiative is designed to facilitate the transition from conventional design tools to more advanced modeling and simulation tools by providing a common user interface for model creation, review, execution, and visualization on personal computers and high-performance computing (HPC) platforms.
+
+The MOOSE MultiApp System allows users to assemble individual codes together to create a customized multiphysics simulation for any reactor design. To couple codes, a hierarchical workflow is invoked involving multiple input files linked together through input syntax that specifies execution order and data transfers between codes. Typically, the MOOSE-based super-application is invoked on an HPC platform. Examples of multiphysics problems specific to nuclear reactor applications are available in the Virtual Test Bed ([VTB](https://mooseframework.inl.gov/virtual_test_bed/)) repository.
+
+Recent work has focused on integrating the NEAMS Workbench with MOOSE-based tools as well as enabling job launch on Idaho National Laboratory (INL) HPC platforms ([Sawtooth](https://hpc.inl.gov/SitePages/Home.aspx)). This integration now permits users to easily run VTB examples (or their own models) using the [*BlueCRAB*](https://www.osti.gov/pages/servlets/purl/1766199)  binary. Access to INL HPC, BlueCRAB and all underlying softwares is granted upon request and appropriate justification through the Nuclear Computational Resource Center ([NCRC](https://inl.gov/ncrc/)). The NEAMS Workbench is available to all INL HPC users automatically.
+
+This documentation aims to provide users with guidelines on how to run VTB examples with the NEAMS Workbench on INL HPC platforms. A *[demonstration video](https://www.youtube.com/watch?v=W6Rq_owHmoU&ab_channel=MarcoDelchini)* is also available (see below) to further illustate all steps of the documentation, and is referenced in the text when needed.
+
+!media neams_workbench/demonstration_video_youtoube.png style=width:80% id=fig:demonstration_video_youtube caption=Demonstration video on how to use the NEAMS Workbench GUI on INL HPC platform.
```

Comment:
I wouldnt have a picture of a video. Users might click on it and feel like they got got

Reply:
I changed it to use the video link directly but the video is not displayed.